src/my_app/test/counter/view/counter_rotary_listener_test.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:async';
+
+import 'package:bloc_test/bloc_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:very_good_wearos_app/counter/counter.dart';
+import 'package:wearable_rotary/wearable_rotary.dart';
+
+import '../../helpers/helpers.dart';
+
+class MockCounterCubit extends MockCubit<int> implements CounterCubit {}
```

Comment:
Should we make it private?

Reply:
gonna keep these elements just like very good core for now.